### DISCUSSION TITLE
스터디 회원이 아니어도 스터디 룸에 들어가도록 허용 해줄까요?

### DISCUSSION THREAD

**Discussion by airman5573** [woowacourse-teams/2022-moamoa#339] [회의 필요]
현재는 스터디 회원이 아니면, 스터디의 소개 페이지만 볼 수 있습니다.  
스터디 룸에 들어가도록 허용 해주는건 어떨까요?  
안에 내용을 궁금해 할것 같아서요

**Comment by tco0427** [comment]
실제로 사용하면서 궁금했었기에 스터디 가입이 안되어있는 상태에서도 조회가 가믕하면 좋을 것 같습니다😁
